game/templates/game/board.html

```
@@ -401,6 +401,10 @@
                 <div style="margin-top: 8px; font-size: 11px; color: rgba(255,255,255,0.5); text-align: center;">
                     Classic · Dark · Green · Blue · Pastel
                 </div>
+                <label style="display:flex;align-items:center;gap:6px;margin-top:10px;font-size:13px;color:#ccc;cursor:pointer;justify-content:center;">
+                    <input type="checkbox" id="showCoordsCheckbox" checked>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)